work-popup.js

```diff
@@ -0,0 +1,131 @@
+    technologies: ['html', 'Ruby on rails', 'css', 'Javascript'],
+    seeLiveLink: '#',
+    seeSourceLink: '#',
+  },
+];
+
+const buttons = document.getElementsByClassName('project_btn');
+const work = document.querySelector('.project');
+const popup = document.createElement('section');
+popup.classList.add('modal');
+function popupDesk(cardNbr) {
+  popup.style.display = 'block';
+  work.appendChild(popup);
+  popup.innerHTML = `
+    <div class="modal-content">
+      <div class="modal-head">
+        <h2>${projects[cardNbr].title}</h2>
+        <i class="fa fa-times" id="popup-close" aria-hidden="true"></i>
+      </div>
+      <ul class="list listPop">
+        <li class="company no_bullet">CANOPY</li>
+        <li class="list_bullet">Back End Dev</li>
+        <li class="list_bullet">2015</li>
+      </ul>
+
+      <div class="modal-image">
+      <img src=${projects[cardNbr].src} alt="App Image">
+      </div>
+      <div class="modal-body">
+        <div class="modal-description">
+          <p>${projects[cardNbr].description[0]}
+        </div>
+        <div class="modal-technologies">
+          <ul class="tag_list no_bullet">
+            <li class="tag">${projects[cardNbr].technologies[0]}</li>
+            <li class="tag">${projects[cardNbr].technologies[1]}</li>
+            <li class="tag">${projects[cardNbr].technologies[2]}</li>
+            <li class="tag">${projects[cardNbr].technologies[3]}</li>
+          </ul>
+          <div class="modal-btns">
+            <button  data-modal-target=".project1" class="project_btn  btn-effect " href="${projects[cardNbr].seeLiveLink}"">See Live </button>
+            <button  data-modal-target=".project1" class="project_btn  btn-effect " href="${projects[cardNbr].seeSourceLink}"">See Source</button>
+          </div>  
+        </div>  
+      </div>;`;
+  document.addEventListener('click', (event) => {
+    if (event.target.id === 'popup-close') {
+      popup.style.display = 'none';
+      document.body.style.overflow = 'scroll';
+    }
+  });
+}
+
+function popUpMobile(cardNbr) {
+  popup.style.display = 'inline';
+  work.appendChild(popup);
+  popup.innerHTML = `
+    <div class="modal">
+    <div class="modal-head">
+      <h2>${projects[cardNbr].title}</h2>
+        <i class="fa fa-times" id="popup-close" aria-hidden="true"></i>
+    </div>
+    <ul class="list listPop">
+      <li class="company no_bullet">CANOPY</li>
+      <li class="list_bullet">Back End Dev</li>
+      <li class="list_bullet">2015</li>
+    </ul>
+    <div class="modal-image">
+      <img src=${projects[cardNbr].src} alt="App Image">
+    </div>
+    <div class="modal-description">
+      <p>${projects[cardNbr].description[0]}</p>
+      <p>${projects[cardNbr].description[1]}</p>
+    </div>
+    <div class="modal-technologies">
+      <ul class="tag_list no_bullet">
```

Comment:
- [ ] **Required:** On the mobile popup, kindly fix the following:
- Some description are undefined
- Add horizontal rule `hr`
- Properly align buttons

Your Implementation | Figma Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/151607594-6e742b5a-2ae7-4c81-80a9-13593054e98d.png) | ![image](https://user-images.githubusercontent.com/27709832/151607668-49a8ba91-fe02-4e18-ad69-2f79eb55c13d.png)

- [ ] **Required:** Kindly align text to the left.

Your Implementation | Figma Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/151607741-cab51f8e-9115-4e9c-b031-21ef60d01da8.png) | ![image](https://user-images.githubusercontent.com/27709832/151607686-95bc28bf-6897-4740-b534-ca59e5773f17.png)

Reply:
- [ ] Some of the changes suggested by @brytebee in the previous review are not yet implemented. Kindly match the popup with the given design by fixing text alignment and buttons as suggested by @brytebee with the screenshots he provided.